### ISSUE TITLE
value too great for base (error token is "10#TIME") #51dded2

### ISSUE THREAD

**Issue by Kasalop** [deajan/osync#164]
**Describe the bug**
After starting osync with a config file the first 4 lines are:
```
/usr/local/bin/osync.sh: line 2141: 10#TIME: value too great for base (error token is "10#TIME")
/usr/local/bin/osync.sh: line 2146: 10#TIME: value too great for base (error token is "10#TIME")
/usr/local/bin/osync.sh: line 2141: 10#conf: value too great for base (error token is "10#conf")
/usr/local/bin/osync.sh: line 2146: 10#conf: value too great for base (error token is "10#conf")
```

**To Reproduce**
1. Setup osync with the following config file / the following parameters
Only changed values are: INSTANCE_ID, INITIATOR_SYNC_DIR=path with dots and underscores but no whitespace, TARGET_SYNC_DIR=remote path with dots and underscores but no whitespaces, SSH_RSA_PRIVATE_KEY, CONFLICT_BACKUP=no, CONFLICT_PREVALANCE=target, SOFT_DELETE=no, SKIP_DELETION=target, DESTINATION_MAILS
2. Run osync with following parameters
`_DEBUG=yes osync.sh /home/osyncuser/sync.conf --dry --verbose --summary`
3. Result
```
/usr/local/bin/osync.sh: line 2141: 10#TIME: value too great for base (error token is "10#TIME")
/usr/local/bin/osync.sh: line 2146: 10#TIME: value too great for base (error token is "10#TIME")
/usr/local/bin/osync.sh: line 2141: 10#conf: value too great for base (error token is "10#conf")
/usr/local/bin/osync.sh: line 2146: 10#conf: value too great for base (error token is "10#conf")
```

**Logs**
The log seems normal apart from these messages. If I append --on-changes I get this additional log messages:
```
/usr/local/bin/osync.sh: line 2141: 10#2019-03-13 19:00:15 - Cannot get value for [CONFIG_FILE_REVISION] in config file [/home/osyncuser/sync > 10#1: syntax error in expression (error token is "19:00:15 - Cannot get value for [CONFIG_FILE_REVISION] in config file [/home/osyncuser/sync > 10#1")
/usr/local/bin/osync.sh: line 2146: 10#2019-03-13 19:00:15 - Cannot get value for [CONFIG_FILE_REVISION] in config file [/home/osyncuser/sync < 10#1: syntax error in expression (error token is "19:00:15 - Cannot get value for [CONFIG_FILE_REVISION] in config file [/home/osyncuser/sync < 10#1")
/usr/local/bin/osync.sh: line 2141: 10#conf: value too great for base (error token is "10#conf")
/usr/local/bin/osync.sh: line 2146: 10#conf: value too great for base (error token is "10#conf")
```

The last two lines are identical, but the first two differ.

**Desktop (please complete the following information):**
 - OS: Not the newest setup: Linux xxx 3.2.0-4-amd64 #1 SMP Debian 3.2.63-2+deb7u2 x86_64 GNU/Linux

I'm not sure if this is a bug in the code or a layer 8 bug in front of the screen ;) I'm new to osync so please be patient if I'm missing something.

**Comment by deajan** [comment]
What version of osync do you run ? Can you post your full anonymized config file ?

**Comment by Kasalop** [comment]
Hi deajan,
it's version of osync 1.3.0-pre-rc1 2019012801 and I linked my config file below.
[osync_sync.conf_ano.txt](https://github.com/deajan/osync/files/2967774/osync_sync.conf_ano.txt)

Thank you for the fast response.

**Comment by deajan** [comment]
You're using an old configuration file from 1.2.x series I suppose, without having run the ugprade script.
In the meantime I've commited a fix in order to make this scenario work, and alert user about the missing config parts.

Can you try last master ?

As a side note, please don't post bad edited conf files like the above, which was probably windows edited since it was not UTF-8 and contained '\r' endings, which makes bug resolution more difficult.

**Comment by Kasalop** [comment]
> 
> 
> You're using an old configuration file from 1.2.x series I suppose, without having run the ugprade script.
> In the meantime I've commited a fix in order to make this scenario work, and alert user about the missing config parts.
works perfectly. Thank you. I compared the two config file versions (mine and from the latest master commit https://github.com/deajan/osync/blob/master/sync.conf.example) and didn't see any differences. Now when I took a second look I saw that it's still ###### osync v1.1x / v2.0x config file rev 2019020801. So I missed this the first time. Sorry for that!

> Can you try last master ?
Works nice :) It mentions that I'm using an old config file. After upgrading the configuration file everything works fine.

> As a side note, please don't post bad edited conf files like the above, which was probably windows edited since it was not UTF-8 and contained '\r' endings, which makes bug resolution more difficult.
I reuploaded the config file in my previous comment. So now it should be utf-8 and should have unix style line endings.

One fast question (I don't want to open another question): How can I update the script to the latest master the right way? I pulled the latest changes and after that I removed the current installation and installed the new one. That works but is it neccessary to reinstall osync after pulling the latest version? I have a server without root previleges. Do I have to contact the admin of this server for every update?
